<launch>
	<param name="serverNameOrIP" value="192.168.2.115" />
		<group ns="head/kinect2">
		<node name="startRGB" pkg="k2_client" type="startRGB"/>
		<node name="startDepth" pkg="k2_client" type="startDepth"/>
		</group>
</launch>
